tests/adminSite/editMenu/coffeeCupAddedToTotal.spec.js

```
@@ -1,5 +1,21 @@
 import { test } from '../../_fixtures/fixtures';
+import { allure } from 'allure-playwright';
 
-test(`New coffee can be added to the  Menu`, async ({}) => {
-  // This is a fake example test.
+
+test.describe('Admin edit menu', () => {
+  test.beforeEach(async () => {
+    await allure.parentSuite('Admin site');
+    await allure.suite('Admin edit menu');
+    await allure.subSuite('Add to menu');
+    await allure.epic('CoffeeCart Admin site');
+    await allure.feature('Edit menu');
+    await allure.severity('critical');
+  });
+
+  test('New coffee can be added to the Menu', async ({ page }) => {
+    await allure.story('User can add new coffee to the menu');
```

Setting `story` inside the test uses the module-level `allure` again. This violates the rule that metadata operations must be tied to the current test: **"The labels must be attached in a way that is tied to the current test via the Playwright Allure adapter API so they appear in the report."**

Also implement a guard such as:

```javascript
if (!testInfo.allure) throw new Error('Allure adapter not configured: ensure the Allure reporter is configured in playwright.config.js');
```

This satisfies: **"HIGH: Do not silently no-op when testInfo.allure is missing. Fail fast or emit a loud error/warning to surface misconfiguration; ensure tests use testInfo.allure.* so labels are correctly attached to the current test."**